bazel/README.md

```diff
@@ -0,0 +1,25 @@
+# Using everst-framework from bazel workspace
```

Comment:
Add something here that bazel is currently best-effort & not tested on CI, but used by Qwello for production usage. Make it clear that bazel related stuff should be pinged to  somebody at Qwello

Reply:
Also, add that c++ module building is currently not implemented yet?

Reply:
No modules are supported in this PR